velox/common/base/tests/ExceptionTest.cpp

```diff
@@ -16,12 +16,19 @@
 
 #include <fmt/format.h>
 #include <folly/Random.h>
+#include <gflags/gflags.h>
 #include <gtest/gtest.h>
 
 #include "velox/common/base/Exceptions.h"
 #include "velox/common/config/GlobalConfig.h"
 #include "velox/flag_definitions/flags.h"
 
+DECLARE_bool(velox_exception_user_stacktrace_enabled);
```

Comment:
@majetideepak @kgpai This change is not right. The flags need to be declared in common header file.

Reply:
@mbasmanova We removed the usage of certain GFlags (including velox_exception_user_stacktrace_enabled) in Velox core in place of GlobalConfig.
Clients such as tests must define GFlags where needed.

Reply:
We can move them to flag_definitions/flags.h for reuse as you suggested. Let me try that.

Reply:
@mbasmanova I was able to move them. Please take another look. Thanks.